deepwell/src/services/page/structs.rs

```diff
@@ -113,6 +119,23 @@ pub struct GetPageOutput {
     pub layout: Layout,
 }
 
+#[derive(Serialize, Debug, Clone)]
+pub struct GetPageDeletedOutput {
+    pub page_id: i64,
+    pub page_created_at: OffsetDateTime,
+    pub page_updated_at: Option<OffsetDateTime>,
+    pub page_deleted_at: Option<OffsetDateTime>,
```

Comment:
By definition, this cannot be `None`.
```suggestion
    pub page_deleted_at: OffsetDateTime,
```